src/tasks/task.spec.ts

```diff
@@ -29,7 +29,8 @@ describe("task", (): void => {
         expect(result.client_ip).toEqual("1.2.3.4");
         expect(result.client_asn).toEqual(10225);
         expect(result.resolver_asn).toEqual(33);
-        const expectedValue = { id: config.tasks[0].id };
+        /* eslint-disable @typescript-eslint/camelcase */
```

Comment:
This directive disables the check for the rest of the file. When the section of "unsafe" code is over, you should have a corresponding `/* eslint-enable @typescript-eslint/camelcase */` directive. Alternatively, at the end of the one line that is problematic, you could simply do `// eslint-disable-line`